src/game_state/Player.cpp

```diff
@@ -59,10 +65,19 @@ void Player::SetYVelocity(float y_velocity) {
 
 void Player::SetSpacebarDown(bool spacebar_down) {
   this->spacebar_down = spacebar_down;
+  if (!this->ground && spacebar_down && this->can_double_jump) {
+    this->is_double_jumping = true;
+    this->can_double_jump = false;
```

Comment:
Ah I see holding the space bar down double jumps for me but only after a few seconds, thigh might just be a input issue though anyway it feels better now that I'm used to it